.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - version-4
+
+permissions: {}
+jobs:
+  release:
+    # prevents this action from running on forks
+    if: github.repository == 'sveltejs/svelte'
+    permissions:
+      contents: write # to create release (changesets/action)
+      pull-requests: write # to create pull request (changesets/action)
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v2.2.4
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: pnpm
+
+      - run: pnpm install --frozen-lockfile
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          # This expects you to have a script called release which does a build for your packages and calls changeset publish
+          publish: pnpm release
```

Comment:
there is no `"release"` script in package.json rn. 

If we don't have to run a script other than changeset publish i think we can remove this line as that is called by default. otherwise https://github.com/sveltejs/kit/blob/master/package.json#L17

Reply:
it looks like I may be wrong. the custom version script is optional, but the publish script seems to be required https://github.com/changesets/action/blob/main/src/index.ts#L43 so even if we don't do custom steps, `"release":"changeset release"` in package.json like in kit repo is needed. Sorry for the confusion.

Reply:
ok. I added that to `package.json` but left out `publish: pnpm release` here. does that sound like it will work?